paper/paper.md

```diff
@@ -378,6 +378,7 @@ following grants:
 - DARPA award HR00112220036
   ([Automating Scientific Knowledge Extraction and Modeling](https://www.darpa.mil/program/automating-scientific-knowledge-extraction-and-modeling))
 - Award number DE-SC0023091 under the DOE Biosystems Design program
+- The PNNL Causality Community of Interest (Causal COIN). The Causal COIN was supported by the Center for AI @pnnl, which serves as a virtual research hub that enables the use of artificial intelligence (AI) across research domains and provided resources supporting the research reported in this article. The Center is driving U.S. leadership in AI and technological innovation by advancing research and development, fostering an AI-enabled workforce, and engaging in impactful industry partnerships.
```

Comment:
I would like to reword this to be more like an acknowledgement. I think updating the sentence above like

> The authors would like to thank the German NFDI4Chem Consortium, PNNL Center for AI, and PNNL Causality Community of Interest (Causal COIN) for support. ...